assets/css/hub-scene.css

```
@@ -0,0 +1,51 @@
+/* Page-scoped sheet for the for-teams hero, so other pages do not ship these
+   rules. See assets/css/finder-hero.css for the @reference/@layer rationale. */
+@reference "tailwindcss";
+@reference "./assets/css/theme.css";
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)